src/components/output/index.tsx

```diff
@@ -68,8 +68,9 @@ export default class App extends Component<Props, State> {
   onRetargetableEvent(event: Event) {
     const targetEl = event.target as HTMLElement;
     if (!this.pinchZoomLeft) throw Error('Missing pinch-zoom element');
-    // If the event is on the handle of the two-up, let it through.
-    if (targetEl.closest('.' + twoUpHandle)) return;
+    // If the event is on the handle of the two-up, let it through,
+    // unless it's a wheel event, in which case always let it through.
+    if (event.type !== 'wheel' && targetEl.closest('.' + twoUpHandle)) return;
```

Comment:
try as I might, I have never managed to remember that closest() exists and accepts selectors